apps/api-gateway/src/board/dto/req.dto.ts

```diff
@@ -0,0 +1,56 @@
+import { PartialType } from '@nestjs/mapped-types';
+import { ApiProperty } from '@nestjs/swagger';
+import { Type } from 'class-transformer';
+import {
+  ArrayMaxSize,
+  IsArray,
+  IsDate,
+  IsNotEmpty,
+  IsNumber,
+  IsOptional,
+  IsString,
+} from 'class-validator';
+
+export class CreateBoardDto {
+  @ApiProperty()
+  @IsString()
+  @IsNotEmpty()
+  public title: string;
+
+  @ApiProperty()
+  @IsString()
+  @IsNotEmpty()
+  public content: string;
+
+  @ApiProperty({ type: 'array', items: { type: 'number' } })
+  @IsArray()
+  @ArrayMaxSize(5)
+  @IsNumber({}, { each: true })
+  @IsNotEmpty()
+  public project_category_id: number[];
+
+  @ApiProperty({ type: 'array', items: { type: 'number' } })
+  @IsArray()
+  @ArrayMaxSize(3)
+  @IsNumber({}, { each: true })
+  @IsNotEmpty()
+  public stack_category_id: number[];
+
+  @ApiProperty({ type: 'number' })
+  @IsNumber()
+  @IsOptional()
+  public position_category_id?: number;
```

Comment:
```suggestion
  @ApiProperty({type : 'number'})
  @IsNumber()
  @IsNotEmpty()
   public project_category_id: number[];

  @ApiProperty({ type: 'array', items: { type: 'number' } })
  @IsArray()
  @ArrayMaxSize(3)
  @IsNumber({}, { each: true })
  @IsNotEmpty()
  public stack_category_id: number[];

  @ApiProperty({ type: 'array', items: { type: 'number' } })
  @IsArray()
  @ArrayMaxSize(5)
  @IsNumber({}, { each: true })
  @IsNotEmpty()
  public position_category_id?: number[];
```

project_category_id 랑 position_category_id 의 명세가 서로 바뀐거 같습니다!

Reply:
수정된 커밋입니다! : 220b93ff7e7ffe5a7f8bfe1ac91e10b6b8582a06